.github/workflows/multi-approvers.yml

```diff
@@ -34,20 +34,29 @@ jobs:
       contains(fromJSON('["pull_request", "pull_request_review"]'), github.event_name)
     runs-on: 'ubuntu-latest'
     steps:
-      - name: 'Checkout'
+      - name: 'Checkout the calling repo'
         uses: 'actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683' # ratchet:actions/checkout@v4
 
+      - name: 'Checkout multi-approvers.js'
+        uses: 'actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683' # ratchet:actions/checkout@v4
+        with:
+          path: 'abcxyz/pkg'
+          repository: 'abcxyz/pkg'
+          sparse-checkout: |
+            .github/workflows/multi-approvers.js
+          sparse-checkout-cone-mode: false
+
```

Comment:
It'd be better just to download the file directly. See how we do this in yaml-lint or go-lint.

Reply:
Thank, Seth. I like this solution a lot better since it avoids potential file collisions as well.

Done.